app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -132,6 +133,25 @@ def update_form_with_classification_code(classification_code)
     invalidate_form_hash
   end
 
+  def log_max_cfi_metrics_on_submission
+    disabilities_with_max_cfi = [ClaimFastTracking::DiagnosticCodes::TINNITUS]
+    if (disabilities_with_max_cfi & diagnostic_codes).none?
+      log_max_cfi_metric_for_disability('none')
```

Comment:
Thinking hard about this case, it might actually be impossible to distinguish between an "educated" user choosing not to check a maxed-out tinnitus, and a user who didn't have tinnitus at all. Since the _submitted_ form's disabilities is just those that they selected. In which case, I think it's fine to not log `none` events at all.